janus_core/md.py

```diff
@@ -846,6 +871,17 @@ def __init__(
             **kwargs,
         )
 
+    def _parameter_prefix(self) -> str:
```

Comment:
The issue is that we want the final state file to use the file prefix but not include the temperature ranges, only the spot temperature, and not the MD temperature, unless I've miss understood @alinelena on the other hand the stats and traj also want to use the file_prefix and the temperature ranges.

Currently if we do that those restart files will include also the temperature range, unless we also have a special file prefix for the final states, which we could. But then we would also need a special function to build that, as we would want the pressure from an NPT but not the T...

Reply:
@alinelena I missunderstodd what @oerc0122 was asking here but this hits at something I am not sure of

NVT_NH inherits from NPT, and therefore inherits the method which puts a pressure in the parameters

But the tests dictate that NVT_NH, should *not* have this pressure listed in the file names CF the tests

[NPT asks for a pressure](https://github.com/stfc/janus-core/blob/main/tests/test_md.py#L168-L172)

[NVT_NH does not, but inhertis from NPT](https://github.com/stfc/janus-core/blob/main/tests/test_md.py#L92-L96), but it will inherit these [_parameter_prefix methods from NPT](https://github.com/stfc/janus-core/blob/main/janus_core/md.py#L804) 

and therefore needs to be "re-overriden"

hence Jacob's question.


Basically should the test change to include the pressure, or should I do what is done here and respect the tests.

Reply:
NVT_NH uses NPT, but sets the barostat time (so `pfactor`) to None (underlying ASE dynamics [here](https://gitlab.com/ase/ase/-/blob/master/ase/md/npt.py?ref_type=heads)), so I don't think we want the pressure labelled, as it's not a meaningful parameter.

Perhaps we should explicitly set `pressure=None` when we call the NPT init via NVT_NH, but I don't think it should make a difference, and it's an extra thing to deal with when setting `externalstress` within NPT.

Reply:
ok great, this is what I thought

I will assume the tests are correct and make sure the pressure is not there

Reply:
be carefull we need to document clear, pressure as target pressure and current pressure that is computed for all ensembles and displayed in -stats